corehq/apps/accounting/views.py

```diff
@@ -338,8 +338,27 @@ def cancel_form(self):
         return CancelForm()
 
     @property
-    def page_context(self):
+    def invoice_context(self):
+        subscriber_domain = self.subscription.subscriber.domain
+
+        invoice_report = InvoiceInterface(self.request)
+        invoice_report.filters.update(subscription__subscriber__domain=subscriber_domain)
```

Comment:
This is not the same as invoices for the subscription - a domain can have different subscriptions at different points in time, so this query can return invoices not from this subscription.  We should fix that, or if that's the intended use case, clarify that in the UI.